Markdownlint fixes

mdo opened the pull request:
Addresses some Markdown linting found from testing out `markdownlint` in #29919.

XhmikosR commented on site/content/docs/5.0/components/card.md:
```diff
@@ -17,7 +17,7 @@ Cards are built with as little markup and styles as possible, but still manage t
 
 {{< example >}}
-<div class="card" style="width: 18rem;">
+<div class="card col-md-6 col-lg-5">
```
These seem unrelated to the patch?

mdo replied:
Oh damn, yes, will fix.

XhmikosR commented:
@mdo did you use my branch https://github.com/twbs/bootstrap/tree/master-xmr-markdownlint? Because there are false positives Markdownlint cannot handle hence why I haven't made a PR to actually add Markdownlint.

mdo commented:
I didn't use your branch—I use markdownlint at GitHub and just ran the CLI against our docs to clear out the easy wins. There are tons of false positives due to our docs tags and what not.

XhmikosR commented on README.md:
```diff
@@ -193,16 +193,16 @@ Thanks to [BrowserStack](https://www.browserstack.com/) for providing the infras
-[![](https://opencollective.com/bootstrap/sponsor/7/avatar.svg)](https://opencollective.com/bootstrap/sponsor/7/website)
-[![](https://opencollective.com/bootstrap/sponsor/8/avatar.svg)](https://opencollective.com/bootstrap/sponsor/8/website)
-[![](https://opencollective.com/bootstrap/sponsor/9/avatar.svg)](https://opencollective.com/bootstrap/sponsor/9/website)
+[![OC sponsor](https://opencollective.com/bootstrap/sponsor/0/avatar.svg)](https://opencollective.com/bootstrap/sponsor/0/website)
```
I'm not sure if "OC Sponsor" is better than nothing. @patrickhlauke

XhmikosR replied:
Let's revert this too @mdo since the patch is only about the dashes for `ul`s.

I will adapt my branch to use dashes too later

patrickhlauke replied:
so i take it there's no way for us to somehow get that pulled in via third party or something and have it provided with more accurate alt etc? (like a validation badge or similar that actually gets generated separately and dynamically pulled in)

if not, then...yeah, better than nothing i guess

XhmikosR replied:
Unfortunately, I'm not aware of a way. But perhaps we should use `OC sponsor 1` etc?

patrickhlauke replied:
i guess you could. that would still not provide the equivalent of what sponsor it is, but i guess it would at least indicate it's not always the same image. least worst fix, probably

XhmikosR commented:
> There are tons of false positives due to our docs tags and what not.

I have disabled some rules in my branch https://github.com/twbs/bootstrap/runs/589127421#step:9:235

mdo committed:
Fix some easy Markdown linting issues

- Always use dashes for bullets
- Add some basic alt text for our readme OC sponsors

mdo commented:
To clarify, this isn't meant to add a script or Action anywhere—it's just cleaning up some Markdown :). I'll defer to another PR for implementing it and running it as a check for us everywhere.

XhmikosR commented:
>To clarify, this isn't meant to add a script or Action anywhere

I know, just trying to get the best of the two branches :)

So, you prefer a `-` for ul style?

That being said, let's wait to hear about the `alt` change from @patrickhlauke. Personally I just ignored these specific errors in README.md in my branch and allowed them everywhere else.

XhmikosR committed:
More Markdownlint fixes

XhmikosR commented:
@patrickhlauke friendly ping :)
